### PR TITLE
Feat: config topology graph

### PR DESCRIPTION
**Pull request by wangyikewxgm**
Signed-off-by: 楚岳 <wangyike.wyk@alibaba-inc.com>

### CODE REVIEW

**Review by wonderflow — COMMENTED** [review]
please make the picture smaller

File: docs/reference/topology-rule.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (barnettZQG) [inline docs/reference/topology-rule.md:1]: How about as an operator manual document?

Comment (wonderflow) [reply docs/reference/topology-rule.md:1]: Yes it should be there

**Review by wonderflow — APPROVED** [review]
i'll update the docs

**Pull request merged**